nightingale_utils/configs/robot_configuration_lookup.yaml

```diff
@@ -29,7 +37,7 @@ joint_configurations: {
             joints: [0.23, 0.23, 0.23],
         },
         right_gripper: {
-            names: ["right_gripper_finger1_joint", "right_gripper_finger2_joint", "right_gripper_finger3_joint" ]
+            names: ["right_gripper_finger1_joint", "right_gripper_finger2_joint", "right_gripper_finger3_joint" ],
```

Comment:
not sure if these should be here but doesn't seem to break anything so we'll keep

Reply:
They are necessary to run the tests, so it's worth keeping here